dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/table/JdbcTeiAnalyticsTableManager.java

```diff
@@ -349,24 +340,20 @@ protected void populateTable( AnalyticsTableUpdateParams params, AnalyticsTableP
 
         removeLastComma( sql )
             .append( " from trackedentityinstance tei" )
-            .append( " left join trackedentitytype tet on tet.trackedentitytypeid = tei.trackedentitytypeid" )
-            .append( " left join programinstance pi on pi.trackedentityinstanceid = tei.trackedentityinstanceid" )
-            .append( " and tei.deleted is false" )
-            .append( " left join program p on p.programid = pi.programid and pi.deleted is false" )
-            .append( " left join programstageinstance psi on psi.programinstanceid = pi.programinstanceid" )
-            .append( " left join programstage ps on ps.programstageid = psi.programstageid" )
```

Comment:
we don't need this joins. Moreover I think the result was a higher number of TEIs with duplicates